kayenta-core/src/main/java/com/netflix/kayenta/canary/CanaryExecutionStatusResponse.java

```diff
@@ -36,6 +36,11 @@
   @NotNull
   protected String pipelineId;
 
+  @NotNull
+  public String getPipelineId() {
```

Comment:
Doesn't this come for free from `@Data`?

Reply:
You'd think so, but there is something very strange happening about protection.  I used `@Getter` which did not allow access, and ended up rolling my own getter for this use case.

Reply:
Ah ha, adding `@Getter` as a default for the class works, even though adding it to individual fields does not.